app-constants.js

```diff
@@ -41,6 +41,7 @@ const requiredEnvVars = [
   'PRODUCT_PROMOS_ENABLED',
   'REDIS_URL',
   'SENTRY_DSN',
+  'SENTRY_DSN_LEGACY',
```

Comment:
See other comment about required vs optional.  I created the "optional" category after many of these vars were already established, so many are in the required column but maybe are more optional 🤷

Reply:
Fixed, thanks!